pkg/cmd/playback.go

```diff
@@ -0,0 +1,169 @@
+package cmd
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+
+	"github.com/spf13/cobra"
+
+	"github.com/stripe/stripe-cli/pkg/playback"
+	"github.com/stripe/stripe-cli/pkg/validators"
+)
+
+const defaultPort = 13111
+const defaultWebhookPort = 13112
+const endpointsDocString = `
+--- Controlling the server ---
+You can configure the running server instance via HTTP GET endpoints (prefixed with "/playback/").
```

Comment:
Can we make these POST?

GETs typically are idempotent and non-effectful -- and these are definitely effectful operations, so as a user of this api I would expect to be instructed to use POST.

Reply:
Good point - fixed in 5321bc7f7d7e1992444b86f205e475a8eced928c!